public/protocols/demo.canvas/protocol.json

```diff
@@ -180,218 +180,6 @@
           "name": "Local Bar"
         }
       ]
-    },
-    "schoolPupils": {
```

Comment:
Is it worth having a development protocol as well so we don't need to recreate the data for testing? Although then we run the risk of having only one protocol be updated when we make changes to the api and not all of them, so maybe not...

Reply:
I think we need a plan for changing the development protocol in feature branches more generally. 

We can discuss this at the kickoff meeting for the next release.

In general, the plan is to have a minimum of two factory protocols: one for developers to test edge cases and new features, and one for the rest of the team and our test sites to have something that somewhat "makes sense", so that they can evaluate and provide feedback.